Assets/UnZENity-Core/Scripts/UI/newMainMenu.cs

```diff
@@ -0,0 +1,72 @@
+using GUZ.Core.Globals;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using ZenKit.Daedalus;
+
+namespace GUZ.Core.UnZENity_Core.Scripts.UI
+{
+    public class newMainMenu : AbstractMenu
```

Comment:
```suggestion
    public class NewMainMenu : AbstractMenu
```

Please use PascalCase. The newMainMenu.prefab should be PascalCased as well.